### PR TITLE
Fixed issue: Incorrect number of hosts down reported

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bugzilla: 1687333
tendrl-bug-id: Tendrl/commons#1076

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat please review

**Comment by GowthamShanmugam** [comment]
There is some problem with fixing testcase issue for tendrl-commons, for now, we will go ahead and merge this PR and i will create an upstream issue for the test case fix.

**Comment by GowthamShanmugam** [comment]
@shtripat Please review and merge this patch

**Comment by GowthamShanmugam** [comment]
Need to fix(todo): Tendrl/commons#1089

File: tendrl/commons/objects/__init__.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def thread_safe_method(self, *args, **kws):
 class BaseObject(object):
     def __init__(self, *args, **kwargs):
         self._ttl = None
+        self._ttl_attr = []
```
Comment (shtripat) [inline tendrl/commons/objects/__init__.py]: Lets rename this attribute as `self._attrs_with_ttl`, looks more meaningful.

Comment (GowthamShanmugam) [reply tendrl/commons/objects/__init__.py]: ok

**Comment by GowthamShanmugam** [comment]
@shtripat verified

**Pull request merged**